### PR TITLE
Skip some acceptance test actions

### PR DESCRIPTION
**Pull request by lego963**

### CODE REVIEW

File: acceptance/openstack/cce/v3/addons_test.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ import (
 )
 
 func TestAddonsLifecycle(t *testing.T) {
+	t.SkipNow()
```
Comment (outcatcher) [inline acceptance/openstack/cce/v3/addons_test.go]: Please add skip reason

File: acceptance/openstack/common.go [hunk 2]
```diff
@@ -17,3 +20,21 @@ func PrintExtension(t *testing.T, extension *extensions.Extension) {
 	t.Logf("Updated: %s", extension.Updated)
 	t.Logf("Links: %v", extension.Links)
 }
+
+func DefaultSecurityGroup(t *testing.T, computeClient *golangsdk.ServiceClient) string {
```
Comment (outcatcher) [inline acceptance/openstack/common.go]: If it requires compute client, maybe create its instance inside of the function?

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: acceptance/openstack/common.go (hunk 2 above)
Comment (lego963) [reply acceptance/openstack/common.go]: solved

File: acceptance/openstack/cce/v3/addons_test.go (hunk 1 above)
Comment (lego963) [reply acceptance/openstack/cce/v3/addons_test.go]: solved

**Review by outcatcher — APPROVED** [review]

**Pull request merged**